runconfig/config.go

```diff
@@ -47,10 +45,8 @@ func DecodeContainerConfig(src io.Reader) (*container.Config, *container.HostCon
 			w.Config.Volumes = make(map[string]struct{})
 		}
 
-		// Now validate all the volumes and binds
-		if err := validateMountSettings(w.Config, hc); err != nil {
-			return nil, nil, nil, err
-		}
+		// With LCOW, we now can't validate mounts before knowing the image platform.
+		// Validation will be done on container creationt
```

Comment:
Outside of the context of this PR this comment isn't going to make any sense to the reader. I think it should be removed.